README.md

```diff
@@ -61,6 +59,190 @@ yarn start
+    docker run -d --name sourcecred --rm --env SOURCECRED_GITHUB_TOKEN -p 8080:8080 sourcecred/sourcecred dev-preview "${REPOSITORY}"
+```
+
+You can also specify an entire organization:
+
+```bash
+ORGANIZATION=@sfosc
+$ SOURCECRED_GITHUB_TOKEN="xxxxxxxxxxxxxxxxx" \
+    docker run -d --name sourcecred --rm --env SOURCECRED_GITHUB_TOKEN -p 8080:8080 sourcecred/sourcecred dev-preview "${ORGANIZATION}"
+```
+
+If you want to bind the data folder to the host, you can do that too. 
+In the example below, we have a folder "data" in the present working directory that we bind to "/data" in the container, the default `SOURCECRED_DIRECTORY`.
+
+```bash
+$ SOURCECRED_GITHUB_TOKEN="xxxxxxxxxxxxxxxxx" \
+    docker run -d --name sourcecred --rm --env SOURCECRED_GITHUB_TOKEN -p 8080:8080 -v $PWD/data:/data sourcecred/sourcecred load "${REPOSITORY}"
+```
+
+We are running in detached mode (-d) so it's easier to remove the container after. 
+After running the command, you can inspect it's progress like this:
+
+```bash
+$ docker logs sourcecred
+  GO   load-sfosc/sfosc
+  GO   github/sfosc/sfosc
+ DONE  github/sfosc/sfosc: 25s
+  GO   compute-cred
+ DONE  compute-cred: 1s
+ DONE  load-sfosc/sfosc: 26s
+...
+```
+
+It will take about 30 seconds to do the initial build, and when the web server is running you'll see this at the end:
+
+```bash
+...
+[./node_modules/react/index.js] 190 bytes {main} {ssr} [built]
+[./src/homepage/index.js] 1.37 KiB {main} [built]
+[./src/homepage/server.js] 5.61 KiB {ssr} [built]
+    + 1006 hidden modules
+ℹ ｢wdm｣: Compiled successfully.
+```
+
+**Important** Although we expose port 0.0.0.0 to be viewable on your host, this is _not a production_ deployment and you should take precaution in how you use it.
+Then you can open up to [http://127.0.0.1:8080](http://127.0.0.1:8080) to see the interface!
+
+![img/home-screen.png](img/home-screen.png)
+
+You can click on "prototype" to see a list of repositories that you generated (we just did sfosc/sfosc):
+
+![img/prototype.png](img/prototype.png)
+
+And then finally, click on the repository name to see the graph.
+
+![img/graph.png](img/graph.png)
+
+When you are finished, stop and remove the container.
+
+```bash
+$ docker stop sourcecred
+```
+
+Since we used the remove (--rm) tag, stopping it will also remove it. 
+If you bound the data folder to the host, you'll see the output remaining there from the generation:
+
+```bash
+$ tree data/
+data/
+├── cache
+│   └── mirror_4d4445774f6c4a6c6347397a61585276636e6b784e546b344f44677a4f54453d.db
+└── projects
+    └── c2Zvc2Mvc2Zvc2M
+        ├── cred.json
+        ├── graph.json
+        └── project.json
+
+3 directories, 4 files
+```
+
+Cool!
+
+
+#### Development Server
+
+If you only want to start the development server, you can run the container like this:
+
+```bash
+$ SOURCECRED_GITHUB_TOKEN="xxxxxxxxxxxxxxxxx" \
+    docker run -d --name sourcecred --rm --env SOURCECRED_GITHUB_TOKEN -p 8080:8080 sourcecred/sourcecred dev-server
```

Comment:
As per previous comment, we _do need_ a data directory here, but _don't need_ a github token. (All data should already be loaded)

```
docker run -d --name sourcecred --rm -p 8080:8080 -v $PWD/data:/data sourcecred/sourcecred dev-server
```

Reply:
This isn't correct - it spits out an error asking for the token.

Reply:
`dev-server` does? That shouldn't be the case.

Reply:
Yeah you're right! I had forgotten to rebuild the container, derp. We're good :)